tests/indexes/tests.py

```diff
@@ -78,12 +100,55 @@ def test_postgresql_text_indexes(self):
         # index (#19441).
         self.assertIn('("slug" varchar_pattern_ops)', index_sql[4])
 
-    @skipUnless(connection.vendor == 'postgresql', "This is a postgresql-specific issue")
-    def test_postgresql_virtual_relation_indexes(self):
+    def test_virtual_relation_indexes(self):
         """Test indexes are not created for related objects"""
         index_sql = connection.schema_editor()._model_indexes_sql(Article)
         self.assertEqual(len(index_sql), 1)
 
+    def test_ops_class(self):
+        index = Index(
+            name='test_ops_class',
+            fields=['headline'],
+            opclasses=['varchar_pattern_ops'],
+        )
+
+        with connection.schema_editor() as editor:
+            editor.add_index(IndexedArticle2, index)
+
+        with connection.schema_editor() as editor:
```

Comment:
Could we check if we actually need this `with` statement again? Maybe the one above is just fine? :man_shrugging: